connectors/src/lib/models.ts

```diff
@@ -0,0 +1,120 @@
+import {
+  type CreationOptional,
+  DataTypes,
+  type ForeignKey,
+  type InferAttributes,
+  type InferCreationAttributes,
+  Model,
+  Sequelize,
+} from "sequelize";
+
+import { type ConnectorProvider } from "../types/connector";
+
+const { CONNECTORS_DATABASE_URI } = process.env;
+
+export const sequelize_conn = new Sequelize(
+  CONNECTORS_DATABASE_URI as string,
+  {}
+);
+
+export class Connector extends Model<
+  InferAttributes<Connector>,
+  InferCreationAttributes<Connector>
+> {
+  declare id: CreationOptional<number>;
+  declare createdAt: CreationOptional<Date>;
+  declare updatedAt: CreationOptional<Date>;
+  declare type: ConnectorProvider;
+  declare nangoConnectionId: string;
+  declare workspaceAPIKey: string;
+  declare workspaceId: string;
+  declare dataSourceName: string;
+}
+
+Connector.init(
```

Comment:
Do we want some unique indexes here ? I'm thinking maybe `(workspaceId, type)`, `(workspaceId, dataSourceName)`